.github/workflows/build.yml

```diff
@@ -10,9 +10,11 @@ on:
   push:
     branches:
       - master
+      - cuda_draft
```

Comment:
maybe we should create a new branch (e.g. `cuda`), as Dan may push to `cuda_draft` directly.

Reply:
I've disabled the GitHub workflow for the CUDA branch since it requires CUDA and won't compile, which always fails.

Reply:
Once this pullrequest is merged, we can create a new branch from cuda_draft and leave cuda_draft for Dan.